_posts/how-do-we-test-and-document-our-c-api.md

```diff
@@ -0,0 +1,420 @@
+---
+title: How do we Test and Document our C API?
+excerpt: 'Wasmer has designed a unique solution to test and document its own C API.'
+date: '2021-07-06T10:32:01.123Z'
+author: Ivan Enderlin
+published: true
+---
+
+[The Wasmer runtime](https://github.com/wasmerio/wasmer) comes in
+multiple flavors. Written in Rust, it is embedded in different
```

Comment:
This will improve SEO for people searching for WebAssembly runtimes in Google.

```suggestion
Wasmer [server-side WebAssembly runtime](https://github.com/wasmerio/wasmer) comes in
multiple flavors. Written in Rust, it is embedded in different
```